docker/docker_container_wrapper.go

```diff
@@ -22,6 +22,14 @@ func (c *dockerContainerWrapper) getExposedTCPPorts() (ports []int) {
 	return
 }
 
+func (c *dockerContainerWrapper) getTags() []string {
+	if tags, exist := c.getEnv()["SRV_TAGS"]; exist {
+		tagsArray := strings.Split(tags, ",")
```

Comment:
Nothing in particular, it is just an array of strings. 
https://github.com/hashicorp/consul/blob/5226e29a691fcf78a034c4f3d249596672a43ac3/command/agent/structs.go#L11